From: =?utf-8?q?Martin_Stegh=C3=B6fer?= <martin@steghoefer.eu>
Date: Tue, 22 Sep 2015 10:17:33 +0200
Subject: Use the float latex package in the spec.

Author: Ralph Giles <giles@mozilla.com>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=790442
Origin: backport, https://git.xiph.org/?p=theora.git;a=commitdiff;h=fbb275803696085225c310773474ef4dcf8a4528

Newer texlive distributions require \usepackage{float} for
the [H] figure layout directive. We use this to make sure
the header layout figures are at the top of their respective
sections for conventional reference like in an RFC, so requiring
the additional package is safer than falling back to standard
latex [h] or [!h].

Note that [H] from the float package is incompatible with
fallback specifiers.

The configure script will automatically check for the new package.

---
 doc/spec/spec.tex | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/doc/spec/spec.tex b/doc/spec/spec.tex
index b29d4ef..db535a4 100644
--- a/doc/spec/spec.tex
+++ b/doc/spec/spec.tex
@@ -11,6 +11,7 @@
 \usepackage{longtable}
 \usepackage{ltablex}
 \usepackage{wrapfig}
+\usepackage{float}
 \usepackage[pdfpagemode=None,pdfstartview=FitH,pdfview=FitH,colorlinks=true]%
  {hyperref}
 
@@ -1949,7 +1950,7 @@ These are indicated as they appear in the sections below.
 \section{Common Header Decode}
 \label{sub:common-header}
 
-\begin{figure}[Htbp]
+\begin{figure}[H]
 \begin{center}
 \begin{verbatim}
   0                   1                   2                   3   
@@ -2024,7 +2025,7 @@ Packets with other header types (\hex{83}--\hex{FF}) are reserved and MUST be
 \section{Identification Header Decode}
 \label{sec:idheader}
 
-\begin{figure}[Htbp]
+\begin{figure}[H]
 \begin{center}
 \begin{verbatim}
   0                   1                   2                   3   
